cmd/cmd/backends.go

```diff
@@ -0,0 +1,149 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	bindAddr string
+	// should port be a string?
+	port                   int
+	tlsConfig              string
+	connectionCost         float64
+	nodeCost               map[string]int
```

Comment:
so this flag is moving from a `nodecost=<JSON dict of string to float64>:` to something like `string=int,string=int,string=int` is this acceptable?

Reply:
same everywhere else we see a `map` type